### PR TITLE
feature/toc 1

### PR DESCRIPTION
**Pull request by jhrr**
[Widgets PR](https://github.com/scaife-viewer/scaife-widgets/pull/17)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "test": "jest --collectCoverage",
     "lint": "vue-cli-service lint",
+    "test": "vue-cli-service test:unit --coverage",
```
Comment (jtauber) [inline package.json]: can you say more about this change?

File: src/App.vue [hunk 2]
```diff
@@ -40,7 +41,7 @@
         ];
       },
       rightWidgets() {
```
Comment (jtauber) [inline src/App.vue]: should be on left-hand side as it's a navigation widget not an information or appearance widget

File: package.json (hunk 1 above)
Comment (jhrr) [reply package.json]: @jtauber yep does exactly the same as before except it prints a coverage report after the test run. It also brings this config into line with the config in `scaife-widgets` which was already using the `vue-cli-service test:unit` plugin.

File: src/App.vue (hunk 2 above)
Comment (jhrr) [reply src/App.vue]: Ok, I will move it across.

File: package.json (hunk 1 above)
Comment (jhrr) [reply package.json]: I can actually see even more low-hanging config fruit that will make the two setups closer to homogeneous, which I think is worth doing to totally eliminate any misleading configuration differences between the two going forward.

Comment (jhrr) [reply package.json]: Seen [this page](https://cli.vuejs.org/core-plugins/unit-jest.html) for reference.  Basically it's a plugin to simplify configuring a Vue project for Jest tests.

Comment (jhrr) [reply package.json]: Jest configurations unified.

**Comment by jacobwegner** [comment]
[As a reader, I can navigate to passages from a TOC widget](https://trello.com/c/NIlYrjXK/49-as-a-reader-i-can-navigate-to-passages-from-a-toc-widget)

**Comment by jacobwegner** [comment]
@jhrr: I stood up #7 doing a Scaife-widgets pin so we could deploy the QA instance:

https://deploy-preview-7--sv-mini.netlify.com/reader?urn=urn%3Acts%3AgreekLit%3Atlg0012.tlg001.perseus-grc2%3A1.1-1.7

since I expect we will have a Scaife widgets release out shortly, no need to do the pin on this particular PR, but keep that in mind in the future (so that what gets deployed to Netlify for the preview deploy is not “lagging” the related Scaife widgets PR).

Thanks for making more things consistent across SW and this package too.

**Pull request merged**